skimage/feature/tests/test_peak.py

```diff
@@ -272,14 +312,14 @@ def test_adjacent_different_objects():
 
 
 def test_four_quadrants():
-    image = np.random.uniform(size=(40, 60))
-    i, j = np.mgrid[0:40, 0:60]
-    labels = 1 + (i >= 20) + (j >= 30) * 2
+    image = np.random.uniform(size=(20, 30))
```

Comment:
OK, I put a seed at the top of each function with random calls. I think that we'll need to setup our tests with a fixture as Stefan said in #2113, but that's another tasks (it impacts several test suites).